L1Trigger/TrackTrigger/plugins/TTStubBuilder.cc

```diff
@@ -41,7 +41,7 @@ void TTStubBuilder< Ref_Phase2TrackerDigi_ >::produce( edm::Event& iEvent, const
   for (const auto& gd : theTrackerGeom->dets())
   {
     DetId detid = (*gd).geographicalId();
-    if(detid.subdetId()!=StripSubdetector::TOB && detid.subdetId()!=StripSubdetector::TID ) continue; // only run on OT
+    if(detid.subdetId()!=TrackerTopology::TOB && detid.subdetId()!=TrackerTopology::TID ) continue; // only run on OT
     if(!tTopo->isLower(detid) ) continue; // loop on the stacks: choose the lower arbitrarily
```

Comment:
not related to the purpose of this PR, but why do we even have TOB and TIB in phase-2 code?

Reply:
I do not know if this is correct (especially since the TrackerGeometryBuilder documentation that @venturia pointed to says that Phase2 only has 4 and 5), but https://github.com/cms-sw/cmssw/blob/master/Geometry/TrackerCommonData/data/PhaseII/trackerParameters.xml defines a scheme for all subdetectors 3, 4, 5 and 6, so at least for the purpose of DetIds (as interpreted by the TrackerTopology) they are defined

Reply:
Whoever modified the code since when I wrote the documentation should have updated the README file consistently. Otherwise we oscillate between no documentation and out-of-date documentation